### ISSUE TITLE
Import YAML creates resources in local cluster instead of downstream cluster

### ISSUE THREAD

**Issue by aaronRancher** [rancher/dashboard#2585]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Bring up a `v2.5.7` Rancher.
Create a downstream cluster.
From downstream cluster, navigate to cluster manager.
Import YAML
![Screen Shot 2021-03-30 at 1 06 13 PM](https://user-images.githubusercontent.com/57409052/113049645-bb45de80-9158-11eb-906e-48d796ed6d40.png)
I used the following YAML to create a deployment.
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: testdeployment
  labels:
    app: testworkload
spec:
  replicas: 1
  selector:
    matchLabels:
      app: mysite
  template:
    metadata:
      labels:
        app: mysite
    spec:
      containers:
        - name: mysite
          image: sangeetha/mytestcontainer
          ports:
            - containerPort: 80
```
Navigate to `Deployments` in Cluster Manager

**Result:**
The deployment is not found. The resource is created in the local cluster instead of the downstream cluster.

**Other details that may be helpful:**
Relevant fix: https://github.com/rancher/rancher/pull/31836

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): `v2.5.7` This is also seen in Rancher `v2.5.6`
- Installation option (single install/HA): Single install

**Comment by vincent99** [comment]
There's nothing for the UI to change here, it's an API fix.

**Comment by aaronRancher** [comment]
Fix for this issue is pending: rancher/rancher#31836

**Comment by gaktive** [comment]
Fix is merged now into master & 2.5-head so please test to see if this is still happening, though do note that at its core, this isn't a UI issue.

**Comment by aaronRancher** [comment]
Validated this fix on Rancher `v2.5-head` `version 2f2cfe6`
Imported resources are created in the downstream cluster, not the local cluster.

Found an issue during testing. The user cannot create secrets or services in a non-default namespace. Will track separately here: https://github.com/rancher/dashboard/issues/2624

Opening an issue to track this fix for `master-head`: https://github.com/rancher/rancher/issues/31922
